crates/cli/Cargo.toml

```diff
@@ -23,6 +23,7 @@ bin-dir = "wasm-bindgen-{ version }-{ target }/{ bin }{ binary-ext }"
 docopt = "1.0"
 env_logger = "0.8"
 anyhow = "1.0"
+fs2 = "0.4.3"
```

Comment:
I assume this was an accidental leftover?

Reply:
I don't remember, I'm going to review that.